lib/OpenQA/Scheduler/Model/Jobs.pm

```diff
@@ -468,24 +468,21 @@ sub incomplete_and_duplicate_stale_jobs ($self) {
     try {
         my $schema = OpenQA::Schema->singleton;
         for my $job ($schema->resultset('Jobs')->stale_ones) {
-            $schema->txn_do(
-                sub {
-                    return $job->reschedule_state if $job->state eq ASSIGNED;
-
-                    my $worker = $job->assigned_worker // $job->worker;
-                    my $worker_info = defined $worker ? ('worker ' . $worker->name) : 'worker';
-                    $job->done(
-                        result => OpenQA::Jobs::Constants::INCOMPLETE,
-                        reason => "abandoned: associated $worker_info has not sent any status updates for too long",
-                    );
-                    my $res = $job->auto_duplicate;
-                    if (ref $res) {
-                        log_warning(sprintf('Dead job %d aborted and duplicated %d', $job->id, $res->id));
-                    }
-                    else {
-                        log_warning(sprintf('Dead job %d aborted as incomplete', $job->id));
-                    }
-                });
+            log_warning(
```

Comment:
Obvious drawback here is that all 3 cases share the same log level. But the message only comes up after the transaction finished successfully if I understand the docs correctly.